src/backend/updates.rs

```diff
@@ -56,6 +57,10 @@ fn post_upgrade() {
     }
     stable_to_heap_core();
     mutate(|state| state.load());
+
+    // TODO: DELETE BEFORE THE NEXT RELEASE
+    mutate(|state| state.memory.set_block_size(1));
```

Comment:
Shouldn't this be the first thing in the `post_upgrade` to ensure that the allocator never sees the block size of 0.